march_hardware_interface/docs/pages/mainpage.dox

```diff
@@ -0,0 +1,45 @@
+/** @mainpage March Hardware Interface
+ * @section introduction Introduction
+ * This package serves as the bridge between the march_hardware package and ros_control.
+ *
+ * Most of this package was directly implemented from
+ *<a href="https://slaterobots.com/blog/5abd8a1ed4442a651de5cb5b/how-to-implement-ros_control-on-a-custom-robot">
+ * this tutorial</a>.
+ * If you want to understand how this package works, it is highly suggested to follow that tutorial first.
+ *
+ * @section features Features
+ * - Parse .yaml files to set configure ros_control limits and controllers.
+ * - Easily switch between different types of controllers for each joint.
+ * - Publish joint states of the exoskeleton.
+ * - Create a command topic for each controller.
+ *
+ * @section how_to_use How to use
+ * To launch, run the following command:
+ * @verbatim roslaunch march_hardware_interface march_controllers.launch @endverbatim
```

Comment:
launching this file will quickly quit due to ethercat not being connected to the computer on which you are testing it. Is it possible to have just the hardware interface running ( with the control manager) so you can load different controllers and check that everything is being registered correctly?

Reply:
Sure you could, but it would defeat the purpose as the whole point is to connect ros_control with the hardware.